dev-app.js

```diff
@@ -55,3 +67,23 @@ app.listen(port, () => {
   console.log("TEST APP TO REDIRECT FOR NEW SESSION : DEV ONLY");
   console.log(`RUNNING ON http://localhost:${port}`);
 });
+
+function getCookieValue(cookie, cookieName) {
+  var value;
```

Comment:
Just a minor observation here but wondering if - since `let` is used elsewhere in similar contexts within the file - it might be preferable to use it here too.